Documentation/kubernetes/compatibility.rst

```diff
@@ -11,16 +11,14 @@ or beta, and may only be available in specific versions of kubernetes.
 ============= =========================== ==========================
  k8s Version   k8s `NetworkPolicy` API      CiliumNetworkPolicy
 ============= =========================== ==========================
- <=1.6         * `extensions/v1beta1`_    `ThirdPartyResource`
+ 1.7           * `extensions/v1beta1`_    `CustomResourceDefinition`
```

Comment:
Collapse all three versions together since they all use the same "k8s NetworkPolicy API" and "CiliumNetworkPolciy".